keyboards/jpe230/big_knob/config.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 jpe230 (@jpe230)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define ENCODER_RESOLUTION 4
```

Comment:
Isnt this the default value?

Reply:
You are right! I'll be removing such definition:

from `encoder.c`:
```
#if !defined(ENCODER_RESOLUTIONS) && !defined(ENCODER_RESOLUTION)
#    define ENCODER_RESOLUTION 4
#endif
```